src/containers/access/src/styled/index.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+const Body = styled.div`
+  display: grid;
+  grid-gap: 36px;
+  grid-auto-flow: row;
+`;
+
+const CenterWrapper = styled.div`
+  align-items: center;
+  display: flex;
+  height: 100%;
+  justify-content: center;
+`;
```

Comment:
I think this is safe to remove since it's only used in `AccessRequestSwitch` to wrap the spinner, for which case I think you can just use `AppContentWrapper` from LUK.